examples/ecs/system_command.rs

```diff
@@ -0,0 +1,32 @@
+use bevy::prelude::*;
+/// This example triggers a system from a command
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(spawn.system())
+        .add_system(trigger_sync.system())
+        .run();
+}
+
+pub struct Player;
+
+/// Spawn some players to sync
+fn spawn(mut commands: Commands) {
+    commands.spawn().insert(Player);
+    commands.spawn().insert(Player);
```

Comment:
Looking at this makes it seem like a copy paste error in my head haha 😅 
Could we change this to be like `for _ in 0..5` or something similar?

Reply:
It is a copy past! from https://github.com/bevyengine/bevy/blob/653c10371e546176059bb779844c0569c0190b6b/examples/ecs/system_param.rs#L33-L35
But I removed one just to be different

Reply:
I still prefer the `for - in` syntax, but I guess I'm ok with how it is.